Tiltfile

```diff
@@ -0,0 +1,93 @@
+load("ext://restart_process", "docker_build_with_restart")
+load("ext://helm_resource", "helm_resource", "helm_repo")
+load("ext://configmap", "configmap_create")
+
+# A list of directories where changes trigger a hot-reload of PATH.
+# Note: this list needs to be updated each time a new package is added to the repo.
+hot_reload_dirs = ["cmd", "config", "gateway", "health", "message", "qos", "relayer", "request", "router"]
+
+# Load the existing config file, if it exists, or use an empty dict as fallback
+localnet_config_path = "localnet_config.yaml"
```

Comment:
Thoughts on renaming  `localnet` to one of the following so it's not the same as a "network"

1. `localgate`
2. `local_gate`
3. `local_gateway`
4. `local`
5. `local_cluster`
6. Other?

My preference is for (1) or (4) but open to pushback to keep `localnet` if you believe it's self explanatory here.

Ditto across the board (variables, filenames, etc.)

Reply:
My vote is for #4 but not strongly attached either.

Reply:
Thanks, renamed to `local`.